plugins/generator-1.18.2/forge-1.18.2/utils/procedures.java.ftl

```diff
@@ -49,11 +49,11 @@
     </#if>
 </#macro>
 
-<#macro procedureOBJToConditionCode object="">
+<#macro procedureOBJToConditionCode object="" defaultValue=true>
     <#if hasProcedure(object)>
         <@procedureToRetvalCode name=object.getName() dependencies=object.getDependencies(generator.getWorkspace()) />
     <#else>
-        true
+        defaultValue

```

Comment:
This is not how printing values in FTL works. You also defined the default value as boolean here, but are using string when calling the methods so this should be uniform

both MC versions